chains/ethereum/config.go

```diff
@@ -20,8 +20,7 @@ type Config struct {
 	endpoint string             // url for rpc endpoint
 	from     string             // address of key to use
 	keystore *keystore.Keystore // Location of keyfiles
-	receiver common.Address
-	emitter  common.Address
+	bridge   common.Address
```

Comment:
I think this should be named `contract`, it's somewhat confusing since this is the bridge